core/translate/mod.rs

```diff
@@ -594,6 +594,10 @@ fn update_pragma(
             query_pragma("journal_mode", header, program)?;
             Ok(())
         }
+        PragmaName::WalCheckpoint => {
+            // TODO
+            Ok(())
```

Comment:
Aah, I see. You might as well implement this in this PR because the infrastructure is there.